index.html

```diff
@@ -549,33 +549,33 @@
             <p>For all PointerEvents in the table above, <a href="https://www.w3.org/TR/uievents/#widl-UIEvent-detail">detail</a> attribute defined in [[!DOM-LEVEL-3-EVENTS]] SHOULD be 0.</p>
 
             <section>
-                <h3><dfn>The <code>pointerover</code> event</dfn></h3>
+                <h3>The <dfn><code>pointerover</code> event</dfn></h3>
```

Comment:
nit: if it's not too much trouble, possible to split out these fixes to `<dfn>` into a separate standalone PR or commit?